scss/_card.scss

```diff
@@ -213,30 +213,3 @@
     }
   }
 }
-
-
-//
-// Accordion
-//
-
-.accordion {
-  overflow-anchor: none;
```

Comment:
I think `overflow-anchor: none;` should be added to `.accordion-button` to avoid reintroducing the issue fixed in https://github.com/twbs/bootstrap/pull/31346